packages/backend/code/index.ts

```diff
@@ -107,11 +123,79 @@ io.on("connection", (socket) => {
   };
   allPlayers.push(newPlayer);
   // Add server listener for revealCell
-  socket.on("revealCell", revealCell);
-  socket.on("becomeSpymaster", becomeSpymaster);
+  socket.on("revealCell", (cellIndex: number, code: string) => {
+    const color = rooms[code].masterBoard[cellIndex].color;
+    rooms[code].publicBoard[cellIndex].color = color;
```

Comment:
Adding this typecast should fix the build.
```suggestion
    rooms[code].publicBoard[cellIndex].color = color as Color;
```